packages/cli/lib/__tests__/projects.test.js

```diff
@@ -1,43 +1,39 @@
+jest.mock('@hubspot/local-dev-lib/logger');
+
 const fs = require('fs');
 const os = require('os');
 const path = require('path');
 const { EXIT_CODES } = require('../enums/exitCodes');
 const projects = require('../projects');
+const { logger } = require('@hubspot/local-dev-lib/logger');
 
 describe('@hubspot/cli/lib/projects', () => {
   describe('validateProjectConfig()', () => {
     let realProcess;
     let projectDir;
     let exitMock;
-    let errorSpy;
 
     beforeAll(() => {
       projectDir = fs.mkdtempSync(path.join(os.tmpdir(), 'projects-'));
       fs.mkdirSync(path.join(projectDir, 'src'));
 
       realProcess = process;
-      errorSpy = jest.spyOn(console, 'error');
```

Comment:
Instead of spying on `console.error`, I just mocked the `@hubspot/local-dev-lib/logger` so this test file doesn't need to be concerned with the implementation details of `logger.error`.  This also has the benefit for keeping the logs from hitting `stderr`